core/src/game_state/play_phase.rs

```diff
@@ -360,6 +362,7 @@ impl PlayPhase {
         landlord: (PlayerID, Rank),
         advancement_policy: AdvancementPolicy,
         max_rank: Rank,
+        lost_on_single_jack: bool,
```

Comment:
this is a bit unclear, I think (_which_ team lost on a single jack?).

Perhaps we can just pass in the last trick entirely?